frontend/app/screens/signin-screen/signin-screen.tsx

```diff
@@ -121,9 +131,9 @@ export const SigninScreen = observer(function SigninScreen() {
                 .finally(() => setSigningIn(false))
             }
           />
-          <Text style={CONT_TEXT}>
+          {/* <Text style={CONT_TEXT}>
```

Comment:
You didn't like this? 😱

Reply:
lol we can add it back !

Reply:
added it back, its a nice touch

Reply:
ahaha I appreciate it, at least I have some contribution to the screen... 😆